src/runtime/hexagon_remote/Makefile

```diff
@@ -57,7 +57,7 @@ CCFLAGS-arm-32-android = --sysroot ${ANDROID_NDK_ROOT}/platforms/android-21/arch
 CCFLAGS-host := ${CCFLAGS} -I ${HEXAGON_TOOLS_ROOT}/include/iss/ -fPIC \
 	-L${HEXAGON_TOOLS_ROOT}/lib/iss/ -lwrapper
 
-CCFLAGS-v60 := $(CCFLAGS-v60) ${COMMON_CCFLAGS} -I ${HEXAGON_SDK_INCLUDES}
+CCFLAGS-v60 := $(CCFLAGS-v60) ${COMMON_CCFLAGS} -I ${HEXAGON_SDK_INCLUDES} -I ${HEXAGON_SDK_ROOT}lib/common/qurt/ADSPv60MP/include
```

Comment:
The 2.0 SDK uses lib, and the 3.0 SDK uses libs.

Same for the inc and incs directories.

 

I do not know why the SDK Folks chose to make this change